lib/wax-unit.js

```diff
@@ -286,6 +154,8 @@ function setContract(account, wasmFile, abiFile) {
     },
     dedupeTapos()
   );
+
+  return new Contract(account, wasmFile, abiJSON, tx);
```

Comment:
Really great that this has the extended functionality yet is (nearly) backward compatible with the previous API